lib/FrameManager.js

```diff
@@ -390,7 +390,7 @@ class Frame {
       const link = document.createElement('link');
       link.rel = 'stylesheet';
       link.href = url;
-      const promise = new Promise(x => link.onload = x);
+      const promise = new Promise(res => link.onload = () => res(link));
```

Comment:
arrow function in arrow function is a little hard to read.

I think this could be written simpler:
```js
async function addStyleUrl(url) {
  const link = document.createElement('link');
  link.rel = 'stylesheet';
  link.href = url;
  document.head.appendChild(link);
  await new Promise(x => link.onload = x);
  return link;
}
```